app/merginapi.h

```diff
@@ -256,6 +257,7 @@ class MerginApi: public QObject
     ProjectStatus getProjectStatus( std::shared_ptr<MerginProject> project, const QDateTime &lastMod );
     QDateTime getLastModifiedFileDateTime( const QString &path );
     int getProjectFilesCount( const QString &path );
+    bool isInIgnore( QFileInfo info );
```

Comment:
it's a good habit to use `const QFileInfo & info` when passing as a function argument